modules/gapi/samples/pipeline_builder_tool.cpp

```diff
@@ -0,0 +1,406 @@
+#include <iostream>
+#include <fstream>
+#include <thread>
+#include <exception>
+#include <unordered_map>
+#include <vector>
+
+#include <opencv2/gapi.hpp>
+#include <opencv2/highgui.hpp> // cv::CommandLineParser
+#include <opencv2/core/utils/filesystem.hpp>
+
+#if defined(WIN32)
```

Comment:
`_WIN32`

https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170

Just `WIN32` is not a compiler definition.